sunpy/sun/models.py

```diff
@@ -18,10 +18,15 @@
 ----------
 * Adapted from Turck-Chieze et al. (1988) with composition: X = 0.7046, Y = 0.2757, Z = 0.0197
 """
```

Comment:
So this top string describes how the code here, so it needs to be updated to acknowledge the increased scope of the code within this module.

Reply:
I am not sure I follow. Can you elaborate please?